compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmDesugarPhase.java

```diff
@@ -160,275 +154,120 @@ private static void rewriteRecordInitFunction(BIRFunction func, BRecordType reco
-        if (function.workerName != null) {
-            function.workerName = getInitialIdString(function.workerName, encodedVsInitialIds);
-        }
-    }
-
-    private static void replaceEncodedAttachedFunctionIdentifiers(List<BAttachedFunction> functions,
-                                                                  HashMap<String, String> encodedVsInitialIds) {
-        for (BAttachedFunction function : functions) {
-            function.funcName = getInitialIdString(function.funcName, encodedVsInitialIds);
-        }
-    }
-
-    private static void replaceEncodedGlobalVariableIdentifiers(List<BIRNode.BIRGlobalVariableDcl> globalVars,
-                                                                HashMap<String, String> encodedVsInitialIds) {
-        for (BIRNode.BIRGlobalVariableDcl globalVar : globalVars) {
-            if (globalVar == null) {
-                continue;
-            }
-            globalVar.name = getInitialIdString(globalVar.name, encodedVsInitialIds);
-        }
-    }
-
-    private static void replaceEncodedImportedGlobalVariableIdentifiers(Set<BIRNode.BIRGlobalVariableDcl> globalVars,
-                                                                        HashMap<String, String> encodedVsInitialIds) {
-        for (BIRNode.BIRGlobalVariableDcl globalVar : globalVars) {
-            globalVar.name = getInitialIdString(globalVar.name, encodedVsInitialIds);
-        }
-    }
-
-    private static String encodeFunctionIdentifier(String identifier, HashMap<String, String> encodedVsInitialIds) {
-        if (encodedVsInitialIds.containsKey(identifier)) {
-            return identifier;
-        }
-        String encodedString = Utils.encodeFunctionIdentifier(identifier);
-        encodedVsInitialIds.putIfAbsent(encodedString, identifier);
-        return encodedString;
-    }
-
-    private static String encodeNonFunctionIdentifier(String identifier, HashMap<String, String> encodedVsInitialIds) {
-        if (encodedVsInitialIds.containsKey(identifier)) {
-            return identifier;
-        }
-        String encodedString = Utils.encodeNonFunctionIdentifier(identifier);
-        encodedVsInitialIds.putIfAbsent(encodedString, identifier);
-        return encodedString;
-    }
-
-    private static String getInitialIdString(String encodedIdString, HashMap<String, String> encodedVsInitialIds) {
-        String initialString = encodedVsInitialIds.get(encodedIdString);
-        if (initialString != null) {
-            return initialString;
-        }
-        return encodedIdString;
-    }
-
-    private static Name getInitialIdString(Name encodedIdString, HashMap<String, String> encodedVsInitialIds) {
-        String initialString = encodedVsInitialIds.get(encodedIdString.value);
-        if (initialString != null) {
-            return Names.fromString(initialString);
-        }
-        return encodedIdString;
+    private static String encodeNonFunctionIdentifier(String identifier) {
```

Comment:
As this function only contains a static method call, we can remove this and use the `Utils.encodeNonFunctionIdentifier(identifier)` directly.